snuba/datasets/sessions.py

```diff
@@ -5,3 +5,8 @@
 class SessionsDataset(Dataset):
     def __init__(self) -> None:
         super().__init__(default_entity=EntityKey.SESSIONS)
+
+
+class OrgSessionsDataset(Dataset):
```

Comment:
You don't need a new dataset for this. The new entity will be part of the sessions dataset.